tests/createWcsWithSip.py

```diff
@@ -79,6 +78,7 @@ def testBigXy0(self):
             src.set(yKey, src.get(yKey) - 500 + y0)
         bbox = afwGeom.Box2I(afwGeom.Point2I(x0, y0), afwGeom.Extent2I(1000, 1000))
         res = self.astrom.determineWcs2(cat, bbox=bbox)
+        self.assertTrue(res.sipWcs is not None, "Failed to fit SIP terms")
```

Comment:
The message will also be clearer since it will tell you that `res.sipWcs` was not none as opposed to telling you that False is not True. I can't think of a reason why we wouldn't want to go for the more explicit test.